### PR TITLE
WIP : Update seldonio/python-builder image

### PR DESCRIPTION
**Pull request by adriangonz**
**What this PR does / why we need it**:

Update `setuptools`, `wheel` and `pip` in `Dockerfile` and remove `flatbuffers`, which are no longer used.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **adriangonz**
You can assign the PR to them by writing `/assign @adriangonz` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: python-builder/Makefile
Comment (axsaucedo) [inline python-builder/Makefile:2]: You'll also need to update the github action test image

**Review by axsaucedo — CHANGES_REQUESTED** [review]

**Comment by adriangonz** [comment]
On 2nd thought, I believe this may not be required anymore. Closing for now.

**Pull request closed without merging**
